config/prod.exs

```diff
@@ -22,5 +22,18 @@ config :logger, level: :info
 
 config :honeybadger, environment_name: :staging
 
+# Configures Oban job processing
+config :dpul_collections, Oban,
+  engine: Oban.Engines.Basic,
+  queues: [cache: 20],
+  repo: DpulCollections.Repo,
+  plugins: [
+    {Oban.Plugins.Cron,
+     crontab: [
+       # Cache Hero Images once a week. 1 AM every Monday.
+       {"0 1 * * MON", DpulCollections.Workers.CacheHeroImages, queue: :cache}
```

Comment:
What is this use case for this schedule? If we were to add or change images I'd think we'd want to cache them at deploy.

Reply:
I'm happy to remove this from a schedule, but I don't think caching every time we deploy is a great idea. That's a lot of unnecessary jobs/calls to the server. Think about how many times we might deploy in a week vs. how often these change. To me, this seemed like a good compromise. Maybe when we run start-over on the indexing consumer?

Reply:
Oh, yeah, I also don't think we want to run this every time we deploy. Only when we deploy a change to the images! I guess if we kept them in a db table we could cache on each save or something? But yes a note in the code makes sense to me!!

Reply:
Done.